packages/auth-helpers/src/lib/session-capability-object.ts

```diff
@@ -0,0 +1,32 @@
+import { SiweMessage } from 'siwe';
```

Comment:
this file contains entry points to getting session capability objects, that are all currently implemented by Recap.